docs/pom.xml

```diff
@@ -37,18 +37,15 @@
                 <artifactId>asciidoctor-maven-plugin</artifactId>
             </plugin>
             <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-assembly-plugin</artifactId>
+            </plugin>
+            <plugin>
                 <groupId>org.codehaus.mojo</groupId>
                 <artifactId>build-helper-maven-plugin</artifactId>
             </plugin>
         </plugins>
     </build>
 
-    <repositories>
```

Comment:
@ricklarsen I believe we need the repository declaration to pull in specific DDF artifacts. I recently added that due to build failures: https://github.com/codice/ddf/commit/03c40ef0ef676faf7047bacc8e0af4d2c831603d. Was this causing an issue?